enter-chroot.sh

```diff
@@ -9,4 +9,17 @@ then
     exit 1
 fi
 
-sudo systemd-nspawn -M homeworld --bind $(pwd):/homeworld:norbind -u "$USER" -a -D "$HOMEWORLD_CHROOT" bash -c "cd /h/ && exec bash"
+ORIG_PWD="$(pwd)"
+cd "$(dirname "$0")"
```

Comment:
It looks like this puts you in a different directory from where you started after the script exits. Is this intended?

Reply:
Huh? The current working directory of the bash subprocess is different from the cwd of the shell.